l10n_br_account/security/ir.model.access.csv

```diff
@@ -35,3 +35,4 @@
 "l10n_br_account_invoice_cce_user","l10n_br_account_invoice_cce_user","model_l10n_br_account_invoice_cce","account.group_account_invoice",1,1,1,0
 "access_account_tax_code_user","access_account_tax_code_user","account.model_account_tax_code","base.group_user",1,0,0,0
 "l10n_br_account_partner_fiscal_type_user","l10n_br_account_partner_fiscal_type_user","model_l10n_br_account_partner_fiscal_type","base.group_user",1,0,0,0
+access_account_invoice_confirm,access_account_invoice_confirm,model_account_invoice_confirm,account.group_account_invoice,1,0,0,0
```

Comment:
Verifica se essa linha é mesmo necessária se a superclasse do `AccountInvoiceConfirm` for `models.TransientModel`